File: platform/android/TrackasiaGLAndroidSDKTestApp/src/main/java/com/trackasia/android/testapp/utils/NavUtils.java
package com.trackasia.android.testapp.utils;

import android.app.Activity;
import android.content.Intent;

import androidx.annotation.NonNull;

import com.trackasia.android.testapp.activity.FeatureOverviewActivity;

public class NavUtils {

  public static void navigateHome(@NonNull Activity context) {
    context.startActivity(new Intent(context, FeatureOverviewActivity.class));
    context.finish();
  }
}

File: platform/android/TrackasiaGLAndroidSDKTestApp/src/main/java/com/trackasia/android/testapp/utils/NavUtils.kt
package com.trackasia.android.testapp.utils

import android.app.Activity
import android.content.Intent
import com.trackasia.android.testapp.activity.FeatureOverviewActivity

object NavUtils {
    fun navigateHome(context: Activity) {
        context.startActivity(Intent(context, FeatureOverviewActivity::class.java))
        context.finish()
    }
}
